ReactWindows/ReactNative/Modules/Image/ImageLoaderModule.cs

```diff
@@ -1,9 +1,42 @@
-﻿using ReactNative.Bridge;
+﻿using Newtonsoft.Json.Linq;
+using ReactNative.Bridge;
+using ReactNative.Views.Image;
+using System;
+using System.Collections.Generic;
+using Windows.ApplicationModel.Core;
+using Windows.Graphics.Imaging;
+using Windows.Storage.Streams;
+using Windows.UI;
+using Windows.UI.Core;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media.Imaging;
 
 namespace ReactNative.Modules.Image
 {
     class ImageLoaderModule : NativeModuleBase
     {
+        private readonly Dictionary<BitmapImage, ExceptionRoutedEventHandler> _imageFailedHandlers =
+            new Dictionary<BitmapImage, ExceptionRoutedEventHandler>();
+
+        private readonly Dictionary<BitmapImage, RoutedEventHandler> _imageOpenedHandlers =
+            new Dictionary<BitmapImage, RoutedEventHandler>();
+
+        private readonly Dictionary<string, BitmapImage> _images =
```

Comment:
This doesn't seem like a good idea to me to keep these images permanently in memory...